PerfettoCds/Pipeline/PerfettoPluginConstants.cs

```diff
@@ -73,67 +76,71 @@ public static class PerfettoPluginConstants
         public const string FtraceEvent = nameof(PerfettoFtraceEvent);
         public const string CpuFrequencyEvent = nameof(PerfettoCpuFrequencyEvent);
         public const string CpuCountersEvent = nameof(PerfettoCpuCountersEvent);
+        public const string GpuCountersEvent = nameof(PerfettoGpuCountersEvent);
         public const string ProcessMemoryEvent = nameof(PerfettoProcessMemoryEvent);
         public const string SystemMemoryEvent = nameof(PerfettoSystemMemoryEvent);
         public const string CpuSamplingEvent = nameof(PerfettoCpuSamplingEvent);
 
         // Paths for source cookers
         public static readonly DataCookerPath SliceCookerPath =
-            new DataCookerPath(PerfettoPluginConstants.ParserId, PerfettoPluginConstants.SliceCookerId);
+            DataCookerPath.ForSource(PerfettoPluginConstants.ParserId, PerfettoPluginConstants.SliceCookerId);
```

Comment:
If you are adding a new func, it seems like PerfettoPluginConstants.ParserId is used everywhere. Maybe put it inside func to reduce code here or at a minimum just make it a param with a default value of this. Also NIT: now that this code is shortened/small I would find it more readable on 1 line per statement (instead of the current 2)

Reply:
I didn't make a new function, this is the new function provided by the SDK.